src/main/webapp/stories.html

```diff
@@ -175,15 +175,30 @@ <h4 class="pb-3 mb-4">Fuse</h4>
               <a href="https://www.redhat.com/en/technologies/jboss-middleware/fuse/" target="_blank">https://www.redhat.com/en/technologies/jboss-middleware/fuse/</a>
             </div>
         </div>
-        
+
         <div class="row mb-5 mt-5">
             <div class="col">
               <h4 class="pb-3 mb-4">Wizzio</h4>
               <p>Digital Banking Platform from Novabase.</p>
               <a href="http://www.novabase.com/financial/wizzio/" target="_blank">http://www.novabase.com/financial/wizzio/</a>
             </div>
         </div>
-        
+
+        <div class="row mb-5 mt-5">
+          <div class="col">
+            <h4 class="pb-3 mb-4">Netflix</h4>
+            <p><i>The Netflix media pipeline processes thousands of new shows and movies every day so that you can watch them on any device anywhere.
```

Comment:
Maybe worth to mention the talk at ApacheCon.

Reply:
@jbonofre right, may be adding the link to the talk schedule?

Reply:
Yes, as soon as the schedule will be online.

Reply:
https://www.apachecon.com/acna19/s/#/scheduledEvent/1209

Reply:
@fpapon mind to update the PR with link to ApacheCon event ?

Reply:
@jbonofre always done on latest force push :)